tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -414,26 +414,26 @@ Feature: antivirus
     Given using <dav-path-version> DAV path
     And user "Brian" has been created with default attributes and without skeleton files
     And user "Alice" has created folder "uploadFolder"
-    And user "Alice" has uploaded file with content "this is a test file." to "uploadFolder/test.txt"
+    And user "Alice" has uploaded file with content "this is a test file1." to "uploadFolder/test1.txt"
     And user "Alice" has shared folder "uploadFolder" with user "Brian" with permissions "all"
-    And user "Alice" has uploaded file with content "this is a test file." to "/test.txt"
-    And user "Alice" has shared file "/test.txt" with user "Brian"
+    And user "Alice" has uploaded file with content "this is a test file2." to "/test2.txt"
+    And user "Alice" has shared file "/test2.txt" with user "Brian"
     And user "Brian" has accepted share "/uploadFolder" offered by user "Alice"
-    And user "Brian" has accepted share "/test.txt" offered by user "Alice"
-    When user "Brian" uploads file with content "X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*" to "Shares/uploadFolder/test.txt" using the WebDAV API
+    And user "Brian" has accepted share "/test2.txt" offered by user "Alice"
+    When user "Brian" uploads file with content "X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*" to "Shares/uploadFolder/test1.txt" using the WebDAV API
     Then the HTTP status code should be "204"
-    And user "Brian" should get a notification with subject "Virus found" and message:
+    And user "Brian" should get a notification for resource "test1.txt" with subject "Virus found" and message:
```

Comment:
Can we keep only this step change without changing file name

Reply:
I changed the file name to demonstrate it works for different names as well, But if it is not required we can change that??

Reply:
@saw-jan This PR motivation is to make our code work for different file names. If we don't implement that then what's the point of this PR

Reply:
I believe that the refactor here was to eliminate false positives which may occur previously if the same name file is in root and in another folder. Since the implementation part is refactored to not let that happen, changing file names should not be necessary.